server/lostandfound/urls.py

```diff
@@ -19,4 +19,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
     url(r'^admin', admin.site.urls),
+    url(r'^admin/', admin.site.urls),
```

Comment:
All it takes is to set it to `True` i think by default it was `False`. That fixes this issue. So no need to have two routes.

It wont make sense to have two routes for each route because of trailing slash.